src/data/nav-items.yaml

```diff
@@ -16,15 +16,15 @@
       path: /getting-started/cli
     - title: Dev-Ops Concepts
       path: /getting-started/dev-ops
-
+      
 - title: Day 0 - Prepare
   pages:
     - title: Overview
       path: /getting-started-day-0
     - title: "Plan the installation"
       path: /getting-started-day-0/plan-installation/ibm-cloud
     - title: "Install with the toolkit"
-      path: /getting-started-day-0/install-toolkit/quick-install
+      path: /getting-started-day-0/install-toolkit/private-catalog
```

Comment:
I think we should leave quick-install as default, and first tab in the install toolkit page